osu.Game/Screens/Play/HUD/BPMCounter.cs

```diff
@@ -37,6 +49,45 @@ protected override void Update()
             Current.Value = GetBPM().Value;
         }
 
+        protected override IHasText CreateText() => new TextComponent();
+
         public bool UsesFixedAnchor { get; set; }
+
+        private class TextComponent : CompositeDrawable, IHasText
+        {
+            public LocalisableString Text
+            {
+                get => text.Text;
+                set => text.Text = value;
+            }
+
+            private readonly OsuSpriteText text;
+
+            public TextComponent()
+            {
+                AutoSizeAxes = Axes.Both;
+
+                InternalChild = new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Horizontal,
+                    Spacing = new Vector2(3, 0),
+                    Children = new Drawable[]
+                    {
+                        text = new OsuSpriteText
+                        {
+                            Font = OsuFont.GetFont(size: 20, weight: FontWeight.Bold),
+                        },
+                        new OsuSpriteText
+                        {
+                            Text = "BPM",
+                            Font = OsuFont.GetFont(size: 14, weight: FontWeight.Bold),
```

Comment:
should be the same text size as the numbers since there isnt two lines of text to fit in

Reply:
I changed size to 20 and 1/2 20